projects/js-packages/number-formatters/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
 	"extends": "jetpack-js-tools/tsconfig.tsc.json",
-	"include": [ "src/index.ts" ],
 	"compilerOptions": {
-		"typeRoots": [ "./node_modules/@types/" ],
-		"sourceMap": false,
-		"outDir": "./dist/",
-		"target": "es2022"
-	}
+		"typeRoots": [ "./node_modules/@types/", "src/*" ],
+		"target": "ES2024",
+		"importHelpers": true, // Import helpers from `tslib` instead of inlining them
+		"customConditions": null // CommonJS build fails when customConditions are defined and this package doesn't need them anyway
```

Comment:
I'm wary of this. the `customConditions` are what makes https://github.com/Automattic/jetpack/blob/978c9d414cd5a6312f4a2e52c2546398bd3fdc79/projects/js-packages/number-formatters/package.json#L36 work.

At the moment, disabling it here is probably ok, since this package doesn't depend on anything else within the monorepo. And maybe this package never will. OTOH, it may be a bad pattern since this may get copied into other packages and break things there.